CanHazFunny/CanHazFunny.Tests/JesterTests.cs

```diff
@@ -1,7 +1,57 @@
 using Xunit;
+using Moq;
+using System;
 
 namespace CanHazFunny.Tests;
 
+
 public class JesterTests
 {
+        [Fact]
+        public void TellJoke_NormalData_SuccessfulReturn()
+        {
+            //Arrange
+            var mockJokeOutput = new Mock<IJokeOutput>();
+            var mockJokeService = new Mock<IJokeService>();
+
+            mockJokeService.SetupSequence(x => x.GetJoke()).Returns("Normal joke");
```

Comment:
```suggestion
            mockJokeService.Setup(x => x.GetJoke()).Returns("Normal joke");
```
`SetupSequence` is not needed here.